.eslintrc.js

```diff
@@ -24,12 +24,13 @@ module.exports = {
   },
   rules: {
     "one-var": ["error", "never"],
+    "no-unused-vars": ["error", { argsIgnorePattern: "^_" }],
   },
   settings: {
     node: {
       allowModules: ["expect-puppeteer"],
       resolvePaths: [__dirname],
-      tryExtensions: [".js", ".json", ".node"],
+      tryExtensions: [".js", ".json", ".node", ".tsx", ".ts"],
```

Comment:
What does this do? 
We don't want `yarn eslint` to run the `eslint` of the `client/src/` files. That's taken care of by `create-react-app`.

Reply:
It's needed so ESLint can resolve some `import` statements in `ssr` for things like `../client/app`

Reply:
Ok. So we keep it.